resources/wallet-generator/monero-wallet-generator.html

```diff
@@ -0,0 +1,8914 @@
+---
+permalink: /generator
```

Comment:
appears when you print. see #2156. an index.html file next to the wallet solves the issue (no need to edit _config.yml anymore)

Reply:
I'm not sure of what you mean. Would you add an empty index.html file?

Reply:
Not a big fan of adding another file for this, but if it's wanted i'm ok with adding it. Leaving this as unresolved in case there is more feedback. Added in the meantime.